lib/Subscribers/ConfirmationEmailMailer.php

```diff
@@ -92,18 +91,12 @@ function sendConfirmationEmail(Subscriber $subscriber) {
     ];
 
     // set from
-    $from = (
-      !empty($signup_confirmation['from'])
-      && !empty($signup_confirmation['from']['address'])
-    ) ? $signup_confirmation['from']
-      : false;
+    $from = $this->settings->get('sender', false);
+    if (empty($from['address'])) $from = false;
```

Comment:
We can delete those 2 lines and also both `$reply_to` lines. Mailer will take care of everything.